Almost all uses of experimental flag are removed

andybak opened the pull request:
1. STL/WRL exports still use the flag. #573 fixes this
2. The experimental setting still handles brushes - this essentially makes the setting an "experimental brush" setting
3. To make room on Labs for 3 new buttons I've hidden Poly, YouTube and Twitch. I *think* these do nothing currently.

mikeage reviewed (commented):
Looks like a great idea and an important step for 3.0.

@mikeskydev , can you comment on the youtube et al buttons

mikeage commented:
Is this waiting for anything (we addressed the streaming in discord, I believe)

andybak commented:
I originally scoped this as "get this out the door before I go on holiday". I realised that shipping (even to beta) when I was about to be unavailable was a bad idea. 

So there's a couple of things I can tweak now I'm not trying to fit it into a single day's work. The wording for the settings button ui can be improved.

And I can document the new settings that arise out of this. I'll do that in the next couple of days.

andybak commented:
These translations are by GPT but it's a fairly simple change so it's hopefully correct.

andybak commented:
So merging this before #573 leaves one small anomaly - STL/WRL exports still rely on the experimental mode setting which the UI claims only affects brushes. 

However I don't think many people care about STL or WRL exports so that is probably fine.

mikeskydev commented:
Would it be possible to now load experimental brushes without restarting the app, by reloading the manifests?

andybak commented:
> Would it be possible to now load experimental brushes without restarting the app, by reloading the manifests?

Yes. I think so. I'll investigate.

Would you prefer that was included in this PR?

mikeskydev commented:
> Would you prefer that was included in this PR?

Yes please!

mikeskydev commented:
Looks like you would need to call `BeginReload()` in BrushCatalog, EnvironmentCatalog etc when updating the manifest. Can see this in App.cs when the initial state load happens.

andybak commented:
@mikeage @mikeskydev Any last thoughts before merging this? There's been some changes since the last review so I wanted to check.

It would be nice to let this sit in beta for a while to shake out any unexpected issues so if possible I'd like to merge it when there's unlikely to be a hotfix or some other reason for us to do an interim release.

mikeage commented:
The fact that we've done three releases in less than 48 hours says nothing about the likelihood of a 4th. 

I'm fine with it from a release management POV. No opinions from a technical POV. But we definitely have a lot of open stuff, and it'd be good to start closing some...
